db/seeds/sandbox_data.rb

```diff
@@ -0,0 +1,17 @@
+require 'faker'
+
+
+Faker::Config.locale = 'en'
+
+providers = ['Capita', 'Teach First', 'UCL', 'Best Practice Network', 'Ambition', 'Education Development Trust']
+providers.each do |provider_name|
+  lead_provider = LeadProvider.find_or_create_by!(name: provider_name)
+  token = LeadProviderApiToken.create_with_random_token!(lead_provider: lead_provider)
+  user_uuids = 10.times.each_with_object([]) do |i, uuids|
+    user = User.create!(full_name: Faker::Name.name, email: Faker::Internet.email)
+    school = School.create!(urn: Faker::Internet.uuid, name: Faker::Company.name, address_line1: Faker::Address.street_address, postcode: Faker::Address.postcode)
+    EarlyCareerTeacherProfile.create!(user: user, school: school)
+    uuids << user.id
+  end
```

Comment:
Can we make it idempotent? I don't know if you plan to run seeding on sandbox often, but it would be nice not to worry about each new seed run adding more fake data.

Reply:
OK, good idea. Although I am not sure whether we want to regenerate them at all or wipe every day and regenerate. Still in discussion

Reply:
I wouldn't regenerate or wipe the profiles, tokens or users, but I would clear down the participation_records on a very frequent (and publicised basis*), since people will be building the client plugins based on these and may need to do work on handling different response codes.

*Cron job or similar set to 10 minutes, or an hour at most.

Reply:
@blackrat might be worth creating a ticket in Jirargh for that for discussion